Library/Homebrew/utils/github.rb

```diff
@@ -337,6 +337,43 @@ def print_pull_requests_matching(query)
     prs.each { |i| puts "#{i["title"]} (#{i["html_url"]})" }
   end
 
+  def fetch_pull_requests(query, tap_full_name, state: nil)
+    issues_for_formula(query, tap_full_name: tap_full_name, state: state).select do |pr|
+      pr["html_url"].include?("/pull/") &&
+        /(^|\s)#{Regexp.quote(query)}(:|\s|$)/i =~ pr["title"]
+    end
+  rescue GitHub::RateLimitExceededError => e
+    opoo e.message
+    []
+  end
+
+  def check_for_duplicate_pull_requests(formula, tap_full_name, version, args, fetch_pr = false)
```

Comment:
```suggestion
  def check_for_duplicate_pull_requests(formula, tap_full_name, version, fetch_pr = false)
```

And use `Homebrew.args` throughout. Alternatively, pass through `force:` and `quiet:` arguments.